rules/windows/process_creation/win_susp_use_of_sqlps_bin.yml

```diff
@@ -0,0 +1,26 @@
+title: Detection of PowerShell Execution via SQL 
+id: 0152550d-3a26-4efd-9f0e-54a0b28ae2f3  
+status: experimental
+description: PowerShell execution through builtin SQL Server "SQLPS.exe" binary.
+references:
+  - https://twitter.com/pabraeken/status/993298228840992768
+  - https://docs.microsoft.com/en-us/sql/tools/sqlps-utility?view=sql-server-ver15 
```

Comment:
Hello @uncleAntik!

Could you please add a link to the relevant [LOLBAS page](https://lolbas-project.github.io/lolbas/OtherMSBinaries/Sqlps/)?